### PR TITLE
45762 Value Added Resellers page

### PR DESCRIPTION
**Pull request by mubeenMastek**

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml [hunk 1]
```diff
@@ -1,199 +1,197 @@
 ﻿@{
     Layout = "~/Views/Shared/_MSILayout.cshtml";
-    ViewData["Header"] = "Value Added Resellers";
+    ViewData["Header"] = "Notice to Mariners";
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml:3]:
```suggestion
    ViewData["Header"] = "Notices to Mariners";
```

**Review by usha13529 — CHANGES_REQUESTED** [review]
One minor comment to address please.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml (hunk 1 above)
Comment (mubeenMastek) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml]: Changes done.

**Review by Chownder — APPROVED** [review]

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @mubeenMastek 

Will merge once live release is complete tomorrow.

**Pull request merged**